### ISSUE TITLE
Add Unit-Tests using JUnit

### ISSUE THREAD

**Issue by kimar** [Geofancy/android-app#10]

**Comment by francisdb** [comment]
Yep, would like to have the fence stuff tested but not sure how feasible this is

**Comment by kimar** [comment]
I'll also take a look into this, still need to get testing implemented by Travis for both Android and iOS (and add lots of tests) before even thinking of streamlining the continous deployment to the store (via maybe fastlane.tools)
